### ISSUE TITLE
Configure the database number with SentinelMonitoredRedisClient ?

### ISSUE THREAD

**Issue by mmagn** [etaty/rediscala#126]
Hi,

How can I configure the db number when creating a SentinelMonitoredRedisClient ?
Is it possible ?

Thanks for your help

**Comment by etaty** [comment]
Yeah you need to make that line https://github.com/etaty/rediscala/blob/ecef67f9e5fa5e2e3b41fe6640d4824dc9b161f1/src/main/scala/redis/Redis.scala#L150 
 a parameter of SentinelMonitoredRedisClient `createClient: (String, Int) => RedisClient`

would you like to create a pull request?

**Comment by mmagn** [comment]
Yes, can you review it ?
https://github.com/etaty/rediscala/pull/127
